crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -244,6 +244,24 @@ fn __roc_force_longjmp(a0: [*c]c_int, a1: c_int) callconv(.C) noreturn {
     longjmp(a0, a1);
 }
 
+fn roc_setjmp_windows_stub(it: [*c]c_int) callconv(.C) c_int {
+    _ = it;
+    return 0;
+}
+
+fn roc_longjmp_windows_stub(a0: [*c]c_int, a1: c_int) callconv(.C) noreturn {
+    _ = a0;
+    _ = a1;
+    std.os.exit(1);
+}
+
+comptime {
+    if (builtin.os.tag == .windows) {
+        @export(roc_longjmp_windows_stub, .{ .name = "longjmp", .linkage = .Strong });
+        @export(roc_setjmp_windows_stub, .{ .name = "setjmp", .linkage = .Strong });
+    }
```

Comment:
`setjmp` and `longjmp` are part of libc, also on windows. Why are these stubs needed?

Reply:
You are correct, I was able to remove these. I am not sure why I added them now.